src/GovUk.Education.ExploreEducationStatistics.Notifier.Tests/Functions/PublicationSubscriptionManagerTests.cs

```diff
@@ -38,37 +38,29 @@ public class SubscriptionManagerFunctionTests(NotifierFunctionsIntegrationTestFi
     public async Task SendsSubscriptionVerificationEmail()
     {
         // Arrange (mocks)
-        var storageTableService = new StorageTableService(Options.Create(new AppSettingsOptions
+        var publicationSubscriptionRepository = new PublicationSubscriptionRepository(Options.Create(new AppSettingsOptions
```

Comment:
How come you can't use `GetRequiredService<IPublicationSubscriptionRepository>()` here? I would have thought it would be set up with the integration test app settings where the connection string has been overriden?

Reply:
Ah, these are the old publication subscription tests - I tried to leave them untouched for now. In these tests, we mock every service, and also mock the settings. So I left this as-is for now, but later it might be good to come back and change these to be in-line with the integration test style we've introduced in this PR for the API subscriptions, at the same time as refactoring all of that code to use the new `DataTableStorageService`?

Reply:
Thanks, that makes sense now. Yes I imagine we'd swap over these tests to use the integration test approach when we do the Publication subscription refactoring.